src/tls_migration.erl

```diff
@@ -0,0 +1,67 @@
+%% TLS migration script (temporary)
+%%
+%% Run from the console as:
+%% > l(tls_migration). tls_migration:main(ok).
+
+-module(tls_migration).
+
+-export([main/1]).
+
+-include_lib("ex_uri/include/ex_uri.hrl").
+-include("logplex_drain.hrl").
+-include("logplex_logging.hrl").
+
+main(_) ->
+    ets:foldl(fun(Drain, _Acc) ->
+                      case should_migrate_drain(Drain) of
+                          true ->
+                              migrate_drain_maybe(Drain);
```

Comment:
Would be nice to export `should_migrate_drain/1` and `migrate_drain_maybe/1` so they can be manually tested in a console session.